test_util/dcos_api_session.py

```diff
@@ -30,10 +30,10 @@ def get_args_from_env():
 
     return {
         'dcos_url': os.environ['DCOS_DNS_ADDRESS'],
-        'masters': os.environ['MASTER_HOSTS'].split(','),
-        'public_masters': os.environ['PUBLIC_MASTER_HOSTS'].split(','),
-        'slaves': os.environ['SLAVE_HOSTS'].split(','),
-        'public_slaves': os.environ['PUBLIC_SLAVE_HOSTS'].split(','),
+        'masters': list(filter(None, os.environ['MASTER_HOSTS'].split(','))),
```

Comment:
A cluster without master is nonsensical (especially in regards to testing), so this should remain unchanged.